Library/Homebrew/formula.rb

```diff
@@ -29,9 +29,12 @@
 # @see https://github.com/styleguide/ruby Ruby Style Guide
 #
 # <pre>class Wget < Formula
+#   desc "Internet file retriever"
 #   homepage "https://www.gnu.org/software/wget/"
-#   url "https://ftp.gnu.org/gnu/wget/wget-1.15.tar.gz"
-#   sha256 "52126be8cf1bddd7536886e74c053ad7d0ed2aa89b4b630f76785bac21695fcd"
+#   url "https://ftpmirror.gnu.org/wget/wget-1.18.tar.xz"
+#   mirror "https://ftp.gnu.org/gnu/wget/wget-1.18.tar.xz"
+#   sha256 "b5b55b75726c04c06fe253daec9329a6f1a3c0c1878e3ea76ebfebc139ea9cc1"
+#   gpg "https://ftpmirror.gnu.org/wget/wget-1.18.tar.xz.sig"
```

Comment:
Updated it because I thought being API documentation it should reflect the type of formula we'd accept, but I'm happy to drop the rest if you like. Agreed on `gpg` for now though & dropped that.